README.md

```diff
@@ -26,7 +26,7 @@ cores. Modin works especially well on larger datasets, where pandas becomes pain
 
 By simply replacing the import statement, Modin offers users effortless speed and scale for their pandas workflows:
 
-<img src="https://github.com/modin-project/modin/raw/7c009c747caa90554607e30b9ac2bd1b190b8c7d/docs/img/Import.gif" style="display: block;margin-left: auto;margin-right: auto;" width="100%"></img>
+<img src="https://github.com/modin-project/modin/raw/master/docs/img/Import.gif" style="display: block;margin-left: auto;margin-right: auto;" width="100%"></img>
```

Comment:
so it would point to the latest gif when we merge the PR

Reply:
That is, any changes for the gif will affect releases in `pypi.org` that use it from the master branch? (for all releases starting from `0.20` release?)

Reply:
> As far as I understand pypi page just renders the current project's readme file regardless of the version.
> 
> So answering your question: yes, all the changes to the gif file will propagate to the pypi page

ok then